content/styles/01-atoms/buttons.css

```diff
@@ -14,7 +14,7 @@ input[type=submit] {
   background-color: var(--color-primary);
   border: 1px solid var(--color-primary);
   border-radius: .3rem;
-  transition: all .3s ease;
+  transition: background-color, color .3s ease;
```

Comment:
Does it work with a space after the coma?

Reply:
I suppose it works. If not we'll fix if we notice.
If we don't notice, it's probably useless ;)

Reply:
I suppose too, I didn't have time to test but the space after the coma was required by my linter...

Reply:
Just tried, it's working 👍